doc/whats_new.rst

```diff
@@ -10,6 +10,39 @@ Version 0.19
 
 **In Development**
 
+Highlights
+----------
+
+We are excited to release a number of great new features including
+:class:`neighbors.LocalOutlierFactor` for anomaly detection,
+:class:`preprocessing.QuantileTransformer` for robust feature
+transformation, and :class:`multioutput.ClassifierChain` to simply
+account for dependencies between classes in multilabel problems. We
+have some new algorithms in existing estimators, such as
+multiplicative update in :class:`decomposition.NMF` and multinomial
+:class:`linear_model.LogisticRegression` with L1 loss.
+
+You can also learn faster.  For instance, the :ref:`new option to cache
+transformations <pipeline_cache>` in :class:`pipeline.Pipeline` makes grid
+search over pipelines including slow transformations much more efficient.  And
+you can predict faster: if you're sure you know what you're doing, you can turn
+off validating that the input is finite using :func:`config_context`.
+
+Cross validation is now able to return the results from multiple metric
```

Comment:
no, it's not. Nor are the TSNE fixes. Being optimisitc, and getting the writing out of the way.